util.go

```diff
@@ -420,3 +420,19 @@ func ReserveSELinuxLabels(store storage.Store, id string) error {
 	}
 	return nil
 }
+
+// IsContainer identifies if the specified container id is a buildah container
+// in the specified store.
+func IsContainer(id string, store storage.Store) (bool, error) {
+	cdir, err := store.ContainerDirectory(id)
+	if err != nil {
+		return false, err
+	}
+	if _, err = ioutil.ReadFile(filepath.Join(cdir, stateFile)); err != nil {
```

Comment:
Is it necessary to read the file, given you're not doing anything with the contents? Feels like this should either try a decode to verify it's a valid Buildah state, or just use os.Stat

Reply:
A stat would probably be enough.